### PR TITLE
Testing collab merge

### PR DESCRIPTION
**Pull request by grizhlieCodes**
Hussain, can you see this?

### CODE REVIEW

**Review by HusJAW — APPROVED** [review]
testing

File: index.html [hunk 1]
```diff
@@ -12,7 +12,13 @@
 </head>
 
 <body>
-  
+  <!-- 
+    1. Create a branch = git branch branchName
+    2. Checkout (move) to the branch = git checkout branchName
```
Comment (HusJAW) [inline index.html:17]: issue !!!

Comment (HusJAW) [reply index.html:17]: there is an issue please review

**Review by HusJAW — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (grizhlieCodes) [reply index.html:17]: No

**Pull request merged**